src/asset-swapper/utils/market_operation_utils/index.ts

```diff
@@ -359,6 +362,7 @@ export class MarketOperationUtils {
         SAMPLER_METRICS.logBlockNumber(blockNumber);
 
         const [makerTokenDecimals, takerTokenDecimals] = tokenDecimals;
+        const isMicroSwap = this.isMicroSwap(makerAmount, ethToMakerAssetRate);
```

Comment:
would ethToMakeAssetRate  be really small for big trades? so we'd want the inverse?

Reply:
discussed offline (trade size doesn't matter for this value)